tests/expressions_case/tests.py

```diff
@@ -119,6 +119,17 @@ def test_annotate_with_join_in_value(self):
             transform=attrgetter('integer', 'join_test')
         )
 
+    def test_annotate_with_in_clause(self):
+        fk_rels = FKCaseTestModel.objects.filter(integer__in=[5])
+        self.assertQuerysetEqual(
+            CaseTestModel.objects.annotate(in_test=Sum(Case(
+                When(fk_rel__in=fk_rels, then=F('fk_rel__integer')),
+                default=Value(0),
+            ))).order_by('pk'),
```

Comment:
To fix the Oracle failure, rephrase this query as
```
            CaseTestModel.objects.only('pk', 'integer').annotate(...
```
or even 
```
            CaseTestModel.objects.values('pk', 'integer').annotate(...
```
I don't have it in me to test myself now, but the issue is that if you don't do one of these, then `annotate()` over `fk_rel` implies `GROUP BY` all the fields of `CaseTestModel`; this model includes BLOBs (`TextField`, `BinaryField`) and Oracle refuses to compare them and so they can't be grouped over.

By the way, doing this should also make your test faster on all databases except PostgreSQL (I think), because PostgreSQL (AFAIK) is the only one smart enough to realize that grouping over all the columns is really the same as grouping over the PK.